rfcs/0001-extended-lookup-tables.md

```diff
@@ -0,0 +1,495 @@
+## Summary
```

Comment:
I'm wondering if the RFC should go into the https://github.com/o1-labs/cryptography-rfcs repo instead?

Reply:
haven't had the time to go through it yet, but I've put mine on https://o1-labs.github.io/mina-book/crypto/plonk/maller_15.html because I could write LaTeX there.

Reply:
Don't know about you guys, but I get a 404 error when trying to access [https://github.com/o1-labs/cryptography-rfcs](https://github.com/o1-labs/cryptography-rfcs)

Reply:
Not sure about you, but I get a 404 error when trying to access the link [ https://github.com/o1-labs/cryptography-rfcs ](https://github.com/o1-labs/cryptography-rfcs )